src/views/Home2/components/FarmsPoolsDataRow.tsx

```diff
@@ -0,0 +1,43 @@
+// import React from 'react'
```

Comment:
Same here , I guess it's not finished, but justify  in case, don't merge commented code

Reply:
ln 16! Apologies - it is nested in a sea of commented code